docs(gen): new trust and safety pages and faq reorder (#5414)
scaleway/docs-content: pages/account/faq.mdx

title: Account FAQ
description: Get answers to your most common account-related questions.
dates:
  validation: 2025-08-18
productIcon: AccountExperienceProductIcon
---

## Getting started

### Where can I find my Organization ID?

> You can find your Organization ID by accessing the [Settings](https://console.scaleway.com/settings/organization/information) page of your Organization.

Reviewer: ```suggestion
You can find your Organization ID in the [Settings](https://console.scaleway.com/settings/organization/information) page of your Organization.
```

You can also click the **Copy ID** button next to the Organization name to do so.

### Can I move a Project to another account?
At this time, it is not possible to move a Project from one account to another.

## Access and security

### My account is locked. What should I do?

> If your account is locked, [contact our Support team](https://console.scaleway.com/support/tickets/create) to learn the reason for the blockage and discover how to regain access.

Reviewer: ```suggestion
If your account is locked, [contact our Support team](https://console.scaleway.com/support/tickets/create) to find out why it was locked and how to regain access.
```

### I cannot access my account, but I am still being billed. How do I resolve this?

> First, you must contact the support to help you [unlock your account](/account/troubleshooting/cannot-log-into-my-account/). Once you have regained access, identify what resources are still running and delete them. You can also [close your account](/account/how-to/close-account), and have all your remaining resources automatically shut down.

Reviewer: ```suggestion
First, you must contact the Support team to help you [unlock your account](/account/troubleshooting/cannot-log-into-my-account/). Once you have regained access, identify what resources are still running and delete them. You can also [close your account](/account/how-to/close-account), and have all your remaining resources automatically shut down.
```

To get started, install the [plugin](https://www.signal-spam.fr/en/nos-modules/) (available for Firefox, Chrome, Safari, and Opera browsers and Mac Mail, Outlook, and Thunderbird mail clients) or follow the official reporting path at [Signal Spam](https://www.signal-spam.fr/en/parcours-signalant/) for further support.

### I don't seem to have access to my Organization's resources. What should I do?
If you do not have access to certain Organization resources, you might not have the necessary [IAM](/iam/concepts/#iam) **permission set**.

[Read the documentation on permission sets](/iam/reference-content/permission-sets/) to learn more about the different access levels.

> Contact an Organization Owner or a Member with the necessary permissions to request rights to the resources you require access to.

Reviewer: ```suggestion
Contact an Organization Owner or a Member with the necessary permissions to request rights to the resources you need to access.
```
